nxdrive/engine/processor.py

```diff
@@ -514,11 +514,7 @@ def _synchronize_direct_transfer(self, doc_pair: DocPair) -> None:
         # Clean-up
         self.dao.remove_state(doc_pair, recursive=False)
 
-        # Display a notification only for files >= 25 MiB (to prevent notifications flood)
-        if not doc_pair.folderish and doc_pair.size >= 25 * 1024 * 1024:
-            self.engine.directTranferStatus.emit(path, False)
```

Comment:
Then `Engine.directTranferStatus` is no more used. Remove all references + add a change note.